wgsl/index.bs

```diff
@@ -8029,6 +8045,13 @@ Each datum is either a [=built-in input value=], or a [=user-defined input datum
 A <dfn noexport>pipeline output</dfn> is a datum the shader provides for further processing downstream in the pipeline.
 Each datum is either a [=built-in output value=], or a [=user-defined output datum|user-defined output=].
 
+[=IO attributes=] are used to establish an object as a [=pipeline input=] or a [=pipeline output=].
+The <dfn noexport>IO attributes</dfn> are:
+* [=attribute/builtin=]
+* [=attribute/location=]
+* [=attribute/interpolate=]
+* [=attribute/invariant=]
```

Comment:
Nit: only builtin and location establish pipeline inputs and outputs. Interpolate and invariant are potential modifiers on established IO. Maybe break this into two lists?